public/css/otc.css

```diff
@@ -32,7 +34,7 @@
   height: 50px;
   opacity: 0;
   pointer-events: none;
-  transition: opacity 0.3s ease-in-out;
+  transition: opacity 0.3s ease-in-out; 
```

Comment:
Can you configure your editor to strip or show trailing whitespace? Would keep changes like these out.